github/Requester.py

```diff
@@ -469,7 +469,7 @@ def __check(
         status: int,
         responseHeaders: Dict[str, Any],
         output: str,
-    ) -> Tuple[Dict[str, Any], Dict[str, Any]]:
```

Comment:
Why removing the `Dict` here?

Reply:
Same above

Reply:
`requestJsonAndCheck` returns what `__check` returns, which is what `__structuredFromJson` returns, which calls into `json.loads(data)`, so should be `Optional[Dict[str, Any]]`.

Reply:
Doesn't `json.loads(data)` actually return `Any`?

Reply:
https://github.com/PyGithub/PyGithub/blob/b6258f4be9bee07beb0935a749289b152b20f7f0/github/PaginatedList.py#L201-L204

Reply:
I'm 100% sure github API not only returns objects

https://api.github.com/repos/minio/minio-js/releases

Reply:
> Right, `Any` still seems a bit broad to me.

But we can't typed it as `list|dict|None`, if you are using a union type, mypy expect all element of this union match expected types.

For example, `list|dict|None` can't be the argument of  `def f(o: list): ...`

Reply:
Awful legacy, well then be it `Any`.

Reply:
Not only legacy problem, call dict method on `None|dict` also raise mypy type checking error.